docs/content/how-to/cdc-ingestion.md

```diff
@@ -692,6 +692,16 @@ you can specify type mapping option `tinyint1-not-bool` (Use `--type-mapping`),
 6. MySQL BINARY will be mapped to Paimon VARBINARY. This is because the binary value is passed as bytes in binlog, so it 
 should be mapped to byte type (BYTES or VARBINARY). We choose VARBINARY because it can retain the length information.
 
+## Adding Properties to a Hive Table
```

Comment:
Don't add here, document this in create-catalogs.md, and for example, after `Synchronizing Partitions into Hive Metastore`.

Reply:
done.